composer.json

```diff
@@ -14,7 +14,8 @@
         }
     ],
     "require": {
-        "guzzlehttp/guzzle": "^7.3"
+        "php": ">=7.2.5",
+        "symfony/http-client": "^5.4"
```

Comment:
will the transition to v 6 require modifications? I ask this because we are soon going to switch to symfony 6 in the core

Reply:
In fact, if we move to Symfony Http Client v6, we need to drop PHP 7.2 version for circuit-breaker too.
I think that could be done in another PR in the future.

Reply:
Yes I didn't think I would do it on the same PR, but it's potentially something to do in the future

Reply:
@jolelievre, not that much, your suggestion can work properly, I will change that today 😉